sample/src/main/java/au/com/jtribe/shelly/sample/MainActivity.java

```diff
@@ -1,134 +1,194 @@
-                        .to("anotheranother@jtribe.com")
-                        .subject("Testing Subject Text")
-                        .body("Testing Body Text")
-                        .send();
-            }
-        });
-
-        findViewById(R.id.email_with_cc_button).setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                Shelly.email(MainActivity.this)
-                        .to("angus@jtribe.com")
-                        .cc("cc@jtribe.com", "anotherccaddress@jtribe.com")
-                        .subject("Testing Subject Text")
-                        .body("Testing Body Text")
-                        .send();
-            }
-        });
-
-        findViewById(R.id.email_with_bcc_button).setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                Shelly.email(MainActivity.this)
-                        .to("angus@jtribe.com")
-                        .cc("cc@jtribe.com", "anotherccaddress@jtribe.com")
-                        .bcc("bcc@jtribe.com", "anotherbcc@jtribe.com")
-                        .subject("Testing Subject Text")
-                        .body("Testing Body Text")
-                        .send();
-            }
-        });
-    }
-
-    @Override
-    protected void onActivityResult(int requestCode, int resultCode, Intent data) {
-        super.onActivityResult(requestCode, resultCode, data);
-
-        if (resultCode == RESULT_OK) {
-            switch (requestCode) {
-                case VIDEO_REQUEST_CODE:
-                    Shelly.share(MainActivity.this)
-                            .text("Testing Video Text")
-                            .video(data.getData())
-                            .send();
-                    break;
-                case IMAGE_REQUEST_CODE:
-                    Shelly.share(MainActivity.this)
-                            .text("Testing Image Text")
-                            .image(data.getData())
-                            .send();
-            }
+    findViewById(R.id.image_and_text_share_button).setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View v) {
+        startActivityForResult(
+            new Intent(Intent.ACTION_PICK, MediaStore.Images.Media.EXTERNAL_CONTENT_URI),
+            IMAGE_REQUEST_CODE);
+      }
+    });
+
+    findViewById(R.id.video_and_text_share_button).setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View v) {
+        startActivityForResult(
+            new Intent(Intent.ACTION_PICK, MediaStore.Video.Media.EXTERNAL_CONTENT_URI),
+            VIDEO_REQUEST_CODE);
+      }
+    });
+
+    findViewById(R.id.email_to_one_button).setOnClickListener(new View.OnClickListener() {
+      @Override
+      public void onClick(View v) {
+        Intent intent = Shelly.email()
+            .to("mitch@jtribe.com.au")
```

Comment:
Maybe don't use your email address in there otherwise you might get spammy test emails. Or a bot will scrape your address from the repo.